src/psyclone/tests/psyGen_test.py

```diff
@@ -914,6 +924,7 @@ def test_globalsum_view(capsys):
     then call view() on that.'''
     from psyclone.psyGen import colored, SCHEDULE_COLOUR_MAP
     from psyclone import dynamo0p3
+    Config.get().api = "dynamo0.3"
```

Comment:
Is this line still needed despite the fixture?

Reply:
Done.